src/components/Table/TableCellSkeleton/TableCellSkeleton.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { FC } from "react";
+import { FC } from "react";
```

Comment:
Isn't it unneeded only starting React 17?

Reply:
@talkor I'm not sure, but the lint yelled at me for it
I think it's from 16
I anyway made a change so it won't yell on it